.travis.yml

```diff
@@ -13,4 +13,5 @@ script:
   - gulp
   - 'test -z "`git status -s | tee /dev/stderr`" || (echo "Encountered untracked changes after generating e2e test. Did you forget run gulp?" && false)'
   - ./e2e_test.sh
+  - ./test_as_depenedency.sh
```

Comment:
This will not work. Travis sets up the environment. Check contents of `e2e_test.sh` vs `run_end_to_end_tests.sh`